.github/workflows/alert-sync-issues.yml

```
@@ -1,14 +1,13 @@
 name: Alert sync issues
-on:
+true:
```

actionlint (1.7.12)

[error] unexpected key "true" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)